xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/AddressCodecTest.java

```diff
@@ -153,6 +155,29 @@ public void encodeDecodeAccountPublicKey() {
     );
   }
 
+  @Test
+  public void addressWithBadChecksum() {
+    Address address = Address.of("r9cZA1mLK5R5am25ArfXFmqgNwjZgnfk59");
+
+    assertThrows(
+      EncodingFormatException.class,
+      () -> addressCodec.classicAddressToXAddress(address, true),
+      "Checksum does not validate"
+    );
+  }
+
+  @Test
+  public void xaddressWithBadChecksum() {
+    XAddress xaddress = XAddress.of("XVLhHMPHU98es4dbozjVtdWzVrDjtV5fdx1mHp98tDMoQXa");
+
+    assertThrows(
+      EncodingFormatException.class,
+      () -> addressCodec.xAddressToClassicAddress(xaddress),
+      "Checksum does not validate"
+    );
+  }
```

Comment:
Sorry, I got confused. I didn't realize `XAddressTest` was in the address codec module. Can you move these back to `XAddressTest` like they were before?